blog/20121227thank-you-first-colo.it.md

```diff
@@ -10,7 +10,7 @@
 
 ![FirstColo](/assets/img/blog/first-color.gif)
 
-[FirstColo](http://www.first-colo.net/EN/), gestore di un moderno data center in Germania,
+[FirstColo](https://www.first-colo.net/EN/), gestore di un moderno data center in Germania,
```

Comment:
https://www.first-colo.net/EN/ shows error 404

I think you can use https://www.first-colo.net/en/ it works